src/usr/local/www/system.php

```diff
@@ -456,7 +456,8 @@ function is_timezone($elt) {
 	'Timezone',
 	$pconfig['timezone'],
 	array_combine($timezonelist, $timezonelist)
-))->setHelp('Select the timezone or location within the timezone to be used by this system.');
+))->setHelp('Select the timezone or location within the timezone to be used by this system. '.
+	'Usually choose a "Continent/City". Only choose a special or "Etc" entry if you understand why you need to use it.');
```

Comment:
Ah, I saw it was deleted and wasn't looking at the commit log, figured you had something else cooking. I've pushed the change now.